app/src/main/java/com/veephealthoutloud/healthoutloud/Garbage.java

```diff
@@ -0,0 +1,7 @@
+package com.veephealthoutloud.healthoutloud;
+
+import java.util.*
+
+public class Garbage {
+    // Hello this is a garbage class. It does nothing ;-;
+}
```

Comment:
Probably should have a new line at the end, is what i would say it this mattered